### PR TITLE
Fix formatting in submission guidelines

### PR DESCRIPTION
**Pull request by sjledoux**
This performs the same fix as #284, just saves them the time of having to fill out the CLA.

### CODE REVIEW

**Review by cfredric — APPROVED** [review]

**Review by Parasos — COMMENTED** [review]
RWS-Submission_Guidelines.md

**Review by Parasos — COMMENTED** [review]
618255c252c4a14f7ddc24f8b6e2bcd132a0760f

**Review by Parasos — COMMENTED** [review]
RWS-Submission_Guidelines.md

File: RWS-Submission_Guidelines.md [hunk 1]
```diff
@@ -102,7 +102,7 @@ A hypothetical example of the RWS canonical list is provided below for reference
 {
   "sets": [
     {
-      "contact": "email address or group alias if available"
```
Comment (Parasos) [inline RWS-Submission_Guidelines.md:105]: RWS-Submission_Guidelines.md

**Review by Parasos — COMMENTED** [review]
RWS-Submission_Guidelines.md

File: RWS-Submission_Guidelines.md (hunk 1 above)
Comment (Parasos) [inline RWS-Submission_Guidelines.md:105]: RWS-Submission_Guidelines.mdRwsCheck.pycheck_sites.py

**Pull request merged**